runtime/common/RecordLogParser.h

```diff
@@ -125,8 +135,13 @@ class BufferHandler {
 
   template <typename T>
   void insertIntoArray(size_t offset, std::size_t index, T value) {
```

Comment:
This would be much more efficient if we used memcpy to copy the data (instead of calling this function once per element) or, in the case of `std::vector<bool>`, used the `std::initializer_list` variant of the `std::vector<bool>` ctor that I detailed in slack.

Reply:
resolved offline